API.md

```diff
@@ -460,7 +454,7 @@ When the server contains exactly one connection, `registrations` is an object wh
 registered plugin name and value contains:
 - `version` - the plugin version.
 - `name` - the plugin name.
-- `options` - options used to register the plugin.
+- `options` - options object passed to the plugin during registration.
```

Comment:
@AdriVanHoudt thanks for the speedy response.  Good question(s).  I will try to answer.  Will take me a few days to get back to you on your suggestions but I will let you know.